src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -88,6 +88,10 @@ static ExperimentalFeature()
                                         source: EngineSource,
                                         isEnabled: false),
                 */
+                new ExperimentalFeature(name: "PSImplicitRemotingBatching",
+                                        description: "Batch implicit remoting proxy commands to improve performance",
+                                        source: "EngineSource",
```

Comment:
This should be `EngineSource` without quotes as it's a const string defined on line 22 of this file

Reply:
Will do.